tests/plugins/autolink/manual/autolink.md

```diff
@@ -1,10 +1,9 @@
 @bender-ui: collapsed
 @bender-tags: 4.7.0, bug, trac14769
-@bender-ckeditor-plugins: clipboard, toolbar, wysiwygarea, elementspath, link, autolink
+@bender-ckeditor-plugins: clipboard, toolbar, wysiwygarea, elementspath, autolink
 
-## Test Scenario
+1. Paste each URL to the editor without trailing spaces.
 
-Paste each URL to the editor and check whether it's turned into a link.
+**Expected** Pasted URL turns into a link.
```

Comment:
Well, I've fallen into my own trap. They say "don't touch old tests!".  And damn, aren't they right.

With the current instructions test fails, as there are URLs that shouldn't turn into links. So here we are adding ourselves work with rephrasing the test which is totally unnecessary as it will not be run in the testing phase anyway (we'll only run manual tests with bender tag in line with the tested release). So as far as I appreciate your effort to improve also this old test, please restore the original phrasing (but you can of course leave `1.` at the beginning).

Reply:
I restored the previous sense of test description.